server/handlers/meshery_pattern_handler.go

```diff
@@ -1276,6 +1276,13 @@ func (h *Handler) DownloadMesheryPatternHandler(
 			http.Error(rw, _errors.Wrapf(err, "failed to export design \"%s\" in %s format", pattern.Name, exportFormat).Error(), http.StatusInternalServerError)
 			return
 		}
+
+		eventDescription := fmt.Sprintf("\"%s\" downloaded as Meshery Design", pattern.Name)
+		event := eventBuilder.WithDescription(eventDescription).Build()
+		err=provider.PersistEvent(event)
+		if(err!=nil){
```

Comment:
Please follow Go idom
```suggestion
		if err!=nil {
```